tripal/src/Services/TripalFileRetriever.php

```diff
@@ -45,16 +52,19 @@ public static function create(ContainerInterface $container) {
   }
 
   /**
-   * Download the contents of a remote or local file from a specified URL,
-   * and returns it in a string variable.
+   * Download the contents of a remote or local file from a specified URL
+   *
+   * The contents of the file is returned in a string variable.
    *
    * @param string $url
    *   The address of the file to download
    * @param array $options
    *   Valid keys:
-   *     retries - int: how many times to retry a download, default = 3
-   *     delay - int: number of seconds between retries, default = 1
-   *     client_options - array: any options to pass to the http client
+   *     retries - int: how many times to retry a download, default = 3.
+   *     rate_limit - float: number of seconds between successive download requests.
+   *     retry_delay - float: number of seconds between retries, default = 1.
+   *     client_options - array: any options to pass to the http client.
```

Comment:
We usually use a list for options so reformatted accordingly.
```suggestion
   *     - retries (int): how many times to retry a download, default = 3.
   *     - rate_limit (float; default = 0): seconds between HTTP requests.
   *     - retry_delay (float; default 1): seconds between retries.
   *     - client_options (array): any options to pass to the http client.
```